### PR TITLE
[ODS-5806] Ed-Fi ODS Docker v1.1.1 for EdFi.Suite3.Ods.WebApi package version 5.1.1 updated v5.1-patch2

### PR DESCRIPTION
**Pull request by semalaiappan**
Note :https://dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_artifacts/feed/EdFi/NuGet/EdFi.Suite3.Ods.WebApi/overview/5.1.1 published to Azure in Release Feed

### CODE REVIEW

File: DB-Admin/Dockerfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ ENV ADMINAPP_DATABASE_VERSION="2.1.1"
 
 COPY init-database.sh /docker-entrypoint-initdb.d/init-database.sh
 
-RUN apk add dos2unix unzip && \
+RUN apk --no-cache add dos2unix=7.4.2-r0 unzip=~6 && \
```
Comment (vimayya) [inline DB-Admin/Dockerfile:19]:
```suggestion
RUN apk --no-cache add dos2unix=~7.4 unzip=~6 && \
```

File: Web-Ods-Api/Dockerfile [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-ENV VERSION="5.1.0"
+ENV VERSION="5.1.1"
 
 WORKDIR /app
 COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 COPY log4net.config /app/log4net.txt
 
-RUN apk add unzip dos2unix bash gettext postgresql-client && \
+RUN apk --no-cache add unzip=~6.0 dos2unix=~7.4 bash=~5.1 gettext=~0.21 postgresql-client=~13.10 icu=~67.1 && \
```
Comment (vimayya) [inline Web-Ods-Api/Dockerfile:16]:
```suggestion
RUN apk --no-cache add unzip=~6.0 dos2unix=~7.4 bash=~5.1 gettext=~0.21 postgresql-client=~13.10 && \
```

File: Web-SwaggerUI/Dockerfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ WORKDIR /app
 COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 
-RUN apk add unzip dos2unix bash gettext && \
+RUN apk --no-cache add unzip=~6.0 dos2unix=~7.4 bash=~5.1 gettext=~0.21 icu=~67.1 && \
```
Comment (vimayya) [inline Web-SwaggerUI/Dockerfile:15]:
```suggestion
RUN apk --no-cache add unzip=~6.0 dos2unix=~7.4 bash=~5.1 gettext=~0.21 && \
```

File: Web-Ods-Api/Dockerfile (hunk 2 above)
Comment (semalaiappan) [reply Web-Ods-Api/Dockerfile]: Fixed it . Thanks

File: Web-SwaggerUI/Dockerfile (hunk 3 above)
Comment (semalaiappan) [reply Web-SwaggerUI/Dockerfile]: Fixed it . Thanks

**Comment by semalaiappan** [comment]
@vimayya  I tested Sandbox build and Shared Instance build . Both are working fine in localhost URL

**Review by vimayya — APPROVED** [review]

**Pull request merged**